tokio-trace/tokio-trace-core/src/subscriber.rs

```diff
@@ -442,4 +451,23 @@ impl Interest {
             _ => false,
         }
     }
+
+    /// More relevant documentation
```

Comment:
?

Reply:
wait... shit

Reply:
Since this is an internal function, i would be fine with just removing this comment if you don't want to write one...